src/isomorphic/classic/element/ReactElementType.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2016-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ * @providesModule ReactElementType
+ */
+
+'use strict';
+
+import type { ReactInstance } from 'ReactInstanceType';
+
+export type Source = {
```

Comment:
I'm not sure if I should put them in this file (which is a bit random) or in a different file